sweeping-the-kitchen.md

```diff
@@ -8,11 +8,11 @@
     . Put them into their place 
     . Start sweeping the kitchen 
 ## *How many people do we need to do this ?* 
-    . At most we need 4 two people 
+    . At most we need 4 people 
     . three people sweep 
     . One moves things like pots 
 ## *How often do we need to do this ?* 
-    . Nedded to do everyday 
```

Comment:
It's good but there is a little bit to improve in your code like "needed to do" should be "needs to be done"